internal/exec/stages/files/files.go

```diff
@@ -87,6 +87,10 @@ func (s stage) Run(config types.Config) error {
 		return fmt.Errorf("creating crypttab entries: %v", err)
 	}
 
+	if err := s.createResultFile(); err != nil {
```

Comment:
Should we do this last, i.e. after relabeling? Makes it clearer that the file should only exist if Ignition succeeded. (Obviously there's other things which could break after that point, like unmounting, but feels like a good intention to have.)

Reply:
Well, I assume we want the result file to be included in the relabeling?  :slightly_smiling_face:

Reply:
Ahh right, yeah that makes sense.